examples/sps/microstructure/sps_elec_2D_YOvac.i

```diff
@@ -214,7 +214,7 @@
     gbindex = 1e6
     surfindex = 1e9
   []
-  # Everything else
+  #  Everything else
```

Comment:
This double space doesn't need to be added here. To avoid triggering the hit format script in the future, please try

```suggestion
  #Everything else
```